modules/rostests/apitests/ntdll/NtAllocateVirtualMemory.c

```diff
@@ -3,6 +3,8 @@
  * LICENSE:         GPLv2+ - See COPYING in the top level directory
  * PURPOSE:         Stress Test for virtual memory allocation
  * PROGRAMMER:      Thomas Faber <thomas.faber@reactos.org>
+ *                  ... <...>
```

Comment:
It's just Timo and Jérôme so you could add them.
Also, please convert to the new SPDX license header, like https://github.com/reactos/reactos/blob/master/modules/rostests/apitests/ntdll/NtFreeVirtualMemory.c#L1-L7

Reply:
Done.